app/views/RoomMembersView/index.tsx

```diff
@@ -277,6 +247,47 @@ const RoomMembersView = (): React.ReactElement => {
 			});
 		}
 
+		if (muteUserPermission) {
+			const { muted = [], ro: readOnly, unmuted = [] } = room;
+			let userIsMuted = !!muted.find?.(m => m === selectedUser.username);
+			let icon: TIconsName = userIsMuted ? 'audio' : 'audio-disabled';
+			let title = I18n.t(userIsMuted ? 'Unmute' : 'Mute');
+			if (compareServerVersion(serverVersion, 'greaterThanOrEqualTo', '6.4.0')) {
```

Comment:
Are you sure this was added in version 6.4.0?
I think the right thing to do is to see when this was implemented.

Reply:
https://github.com/RocketChat/Rocket.Chat/pull/30050